README.md

```diff
@@ -102,8 +102,9 @@ definition of OpenShift in some way, add a new required dependency, or
 change the way customers are supported.  Use your best judgement to
 determine the level of approval needed.  If you’re not sure, ask a
 staff engineer to help find a good approver by posting in
-`#forum-arch` on the CoreOS Slack server and tagging
-`@aos-staff-engineers`.
+`#forum-arch` on the RedHat Slack server and tagging
+`@aos-staff-engineers`.  If you are external to RedHat, you can use the
+#openshift-users forum on the kubernetes.slack.com instance.
```

Comment:
```suggestion
`#openshift-users` forum on the kubernetes.slack.com instance.
```

The linter is interpreting '^#' as starting a heading.

Reply:
thanks, fixed